### ISSUE TITLE
Add schema slots for GNSP records and JGI study identifiers to data portal study pages

### ISSUE THREAD

**Issue by aclum** [microbiomedata/nmdc-server#971]
Mark is working on two new identifiers, one for GNSP task records and one for JGI genome portal study pages, 

 that need to be displayed on the data portal study page. 
The CURIES for these identifiers are registered at bioregistry.io (not identifiers.org). 

These new identifiers need to be linked to the study page under external resources.

Depends on https://github.com/microbiomedata/nmdc-schema/issues/868

cc @shreddd

**Comment by ssarrafan** [comment]
@aclum I assigned this to Mark. If not accurate can you please re-assign?

**Comment by aclum** [comment]
No, this is a data portal issue so should be assigned to Mike.

**Comment by aclum** [comment]
backlog pending data portal study page review squad.

**Comment by aclum** [comment]
@mslarae13 @lamccue were these study slots covered in the study page refresh squad? We would still like [gnps_task_identifiers](https://microbiomedata.github.io/nmdc-schema/gnps_task_identifiers/) and [jgi_portal_study_identifiers](https://microbiomedata.github.io/nmdc-schema/jgi_portal_study_identifiers/) displayed on the study pages.

**Comment by lamccue** [comment]
No, we did not look at slots that did not already have values being displayed on the study pages.

**Comment by cthoyt** [comment]
Is there anything else you need from the Bioregistry side?

**Comment by aclum** [comment]
no, nothing else is needed from bioregistry @cthoyt thanks!

**Comment by mslarae13** [comment]
I _think_ jgi_portal_study_identifiers is being displayed.
The prefix is jgi.proposal: & we have the JGI proposal DOI shared. Isn't this basically the same thing? Or are you saying you want that 5 digit ID shared?

For EMSL, we settled on just the DOI cuz the DOI has the ID and it's redundant because the DOI provides the ID.

**Comment by aclum** [comment]
This issue is still outstanding. The study refresh squad did not address any slots that weren't already being displayed per a slack message with Lee Ann.

We(I) want the 5 digit ID shared as we can't interoperate between them without calling the OSTI API. 

The GNPS identifiers are also still not being displayed.
